usaspending_api/search/v2/views/search.py

```diff
@@ -497,118 +243,4 @@ def post(self, request):
                 result_key = categories[award[category_name]]
             results[result_key] += award['category_count']
 
-        # build response
-        return Response({"results": results})
-
-    #  ###############################  #
```

Comment:
Moved these classes to `usaspending_api/search/v2/views/search_elasticsearch.py`